MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift

```diff
@@ -395,15 +397,15 @@ public class MUXSDKPlayerBinding: NSObject {
         self.videoData.hasUpdates = false
     }
     
-    private func handleRenditionChange(bitrate: Double) {
-        guard self.videoData.lastAdvertisedBitrate != 0 else {
-            // Starting Playback
+    private func handleRenditionChange(bitrate: Double, playbackStartDate: Date?) {
+        guard
+            self.videoData.lastAdvertisedBitrate != 0,
+            playbackStartDate != nil
```

Comment:
I dont think we can rely on this for the first rendition since we can get other rendition changes with `playbackStartDate == nil` later that just won't be played. I think for the first one we essentially want to make sure to get the last one before the first playback rate change and then no more.